### PR TITLE
Update accessibility.md

### PR DESCRIPTION
**Pull request by nathanlschneider**
Fixed grammar issue.

### CODE REVIEW

File: content/docs/accessibility.md [hunk 1]
```diff
@@ -369,7 +369,7 @@ class BlurExample extends React.Component {
 ```
 
 This code exposes the functionality to both pointer device and keyboard users. Also note the added `aria-*` props to support screen-reader users. For simplicity's sake
-the keyboard events to enable `arrow key` interaction of the popover options have not been implemented.
+the keyboard events to enable `arrow key` interaction of the popover options has not been implemented.
```
Comment (AlmeroSteyn) [inline content/docs/accessibility.md]: This refers to the keyboard events and should, I think, be `have` for plural.

File: content/docs/accessibility.md [hunk 2]
```diff
@@ -306,7 +306,7 @@ as the `window` object never receives a `click` event. This can lead to obscured
 
 <img src="../images/docs/outerclick-with-keyboard.gif" alt="A toggle button opening a popover list implemented with the click outside pattern and operated with the keyboard showing the popover not being closed on blur and it obscuring other screen elements." />
 
-The same functionality can be achieved by using an appropriate event handlers instead, such as `onBlur` and `onFocus`:
+The same functionality can be achieved by using an appropriate event handler instead, such as `onBlur` and `onFocus`:
```
Comment (AlmeroSteyn) [inline content/docs/accessibility.md]: Yeah this is wrong. In the example below this, both `onFocus` and `onBlur` have been implemented, so perhaps `using appropriate event handlers` will be better.

**Comment by tesseralis** [comment]
Closing because as @AlmeroSteyn said, the current grammar is correct.

**Comment by AlmeroSteyn** [comment]
@tesseralis The one sentence is indeed a little broken:

> The same functionality can be achieved by using an appropriate event handlers instead, such as `onBlur` and `onFocus`:

should become 

The same functionality can be achieved by using an appropriate event handler instead, such as `onBlur` and `onFocus`:

So let's re-open :-)

**Comment by tesseralis** [comment]
@AlmeroSteyn oh, you're right, I didn't see that.

My preference would be

> The same functionality can be achieved by using the appropriate event handlers instead, such as `onBlur` and `onFocus`:

**Comment by AlmeroSteyn** [comment]
@tesseralis Yeah you are quite right! Also along the lines of what I suggested initially. Don't comment from your phone, haha!

@nathanlschneider Will you be able to update?

**Pull request closed without merging**